src/commands/dev/mod.rs

```diff
@@ -89,7 +114,16 @@ fn preview_request(
     let now: DateTime<Local> = Local::now();
     let preview_id = &preview_id;
 
-    parts.uri = get_preview_url(&path).expect("Could not get preview url");
+    let mut headers: HeaderMap = HeaderMap::new();
```

Comment:
maybe factor this header prefix addition logic out into another function that acts as an "opposite" function to `munge_response_headers`?